packages/auth/src/accessToken/routes.ts

```diff
@@ -1,26 +1,27 @@
 import { Logger } from 'pino'
-import { ActiveTokenInfo, TokenInfo } from 'token-introspection'
-import { Access } from '../access/model'
+import { TokenInfo } from 'token-introspection'
+import { toOpenPaymentsAccess } from '../access/model'
 import { AppContext } from '../app'
 import { IAppConfig } from '../config/app'
 import { AccessTokenService } from './service'
-import { accessToBody } from '../shared/utils'
 import { ClientService } from '../client/service'
 import { Grant } from '../grant/model'
+import { toOpenPaymentsAccessToken } from './model'
+import { AccessService } from '../access/service'
 
-type TokenRequest<BodyT> = Omit<AppContext['request'], 'body'> & {
+type TokenRequest<BodyT> = Exclude<AppContext['request'], 'body'> & {
```

Comment:
Similar to the `backend` package, using `Exclude` instead of `Omit` properly gives us typing for any of the context types that extended `AppContext`